optimum/intel/neural_compressor/modeling_base.py

```diff
@@ -164,53 +281,6 @@ def _from_pretrained(
                 )
 
         model_save_dir = Path(model_cache_path).parent
-
-        if is_itrex_available():
```

Comment:
we can leave a warning to state this is now deprecated

Reply:
yes, I add the deprecated when do woq quantization and loading woq weights.